src/include/duckdb/transaction/duck_transaction.hpp

```diff
@@ -23,6 +24,11 @@ class StorageCommitState;
 struct DataTableInfo;
 struct UndoBufferProperties;
 
+struct CommitInfo {
+	transaction_t commit_id;
+	ActiveTransactionState active_transactions = ActiveTransactionState::UNSET;
```

Comment:
`ActiveTransactionState active_transactions = ActiveTransactionState::UNSET;` seeing it in use like that I'm wondering if we want to call it `UNKNOWN` instead of `UNSET`?